services/exportService.ts

```diff
@@ -71,39 +68,38 @@ export async function importFeedFromFile(jsonFromFile: string) {
   }
 }
 
-
 function refactorExportToFeed(feed: TransferFeed): SiteFeed {
-  const url = feed.domain + feed.url;
-
-  const visited =
-    feed.visited?.reduce<Record<string, boolean>>((acc, path) => {
-      acc[feed.domain + path] = true;
-      return acc;
-    }, {}) || {};
-  return { url: url, visited: visited };
+  const { domain } = feed;
+  const url = domain + feed.url;
+
+  const visited = (feed.visited ?? []).reduce((acc, path) => {
+    return { ...acc, [domain + path]: true };
+  }, {});
+
+  return { url, visited, favorite: feed.favorite ?? false };
```

Comment:
https://www.richsnapp.com/article/2019/06-09-reduce-spread-anti-pattern

maybe we should not use this pattern here